src/nimlsp.nim

```diff
@@ -123,7 +124,7 @@ type Certainty = enum
   Nimble
 
 proc getProjectFile(file: string): string =
-  result = file
+  result = decodeUrl(file)
```

Comment:
I guess this would work fork Windows paths like `file:///C:/foo/bar` as well? If so maybe we can remove the ugly fix on the following lines.

Reply:
Ah, never mind this, I looked up what decodeUrl does and this seems like a good idea. Should it be applied to other things as well?

Reply:
https://play.nim-lang.org/#ix=2nhB

Reply:
other things? I just notice two place need this, one is uri to path for getting nimsuggest, one is initial nimsuggest put project file path.